airbyte-integrations/connectors/source-salesforce/source_salesforce/streams.py

```diff
@@ -298,7 +304,7 @@ def _read_pages(
         # Always return an empty generator just in case no records were ever yielded
         yield from []
 
-    @default_backoff_handler(max_tries=5, backoff_method=backoff.constant, backoff_params={"interval": 5})
+    @default_backoff_handler(max_tries=5)  # FIXME remove once HttpStream relies on the HttpClient
```

Comment:
I'm fine with changing that behavior from expo to constant as most of the errors should be retried on the HttpStream level. I'm not too concerned based on the cases I've seen in the past that `constant` would perform worse than `expo` but I'm willing to re-evaluate that